projects/05-social-network-fw/README.md

```diff
@@ -51,11 +51,9 @@ cero, su alcance, sus particularidades, el flujo, las validaciones, etc.
 Concéntrate en aprender y utilizar estas nuevas tecnologías.
 
 El alcance del proyecto y el detalle de sus características es exactamente
-[el mismo que el original](https://github.com/Laboratoria/curricula-js/tree/v2.x/projects/03-social-network),
+[el mismo que el original](https://github.com/Laboratoria/curricula-js/tree/main/projects/03-social-network),
```

Comment:
are these links updated correctly? `v2.x` was broken. I had to change this a few times.

Reply:
That's right. Well spotted!